controllers/operatorpolicy_status.go

```diff
@@ -1059,10 +1104,22 @@ func existingInstallPlanObj(ip client.Object, phase string) policyv1.RelatedObje
 	}
 
 	switch phase {
+	case string(operatorv1alpha1.InstallPlanPhaseInstalling):
+		// Check policy.spec.statusConfig.upgradesAvailable to determine `compliant`.
+		if complianceConfig != "Compliant" {
+			relObj.Compliant = string(policyv1.NonCompliant)
+		} else {
+			relObj.Compliant = string(policyv1.Compliant)
+			relObj.Reason += " (policy compliance is not impacted due to spec.complianceConfig.upgradesAvailable)"
```

Comment:
I find this message to be quite noisy for a default behavior. My thought is that we only need to include a message such as this when the default is overridden.

Reply:
@JustinKuli what do you think?

Reply:
I don't think it's prohibitively noisy, but I see what you mean with default behaviors. So then should we have a message on the NonCompliant case? Maybe "(in violation due to spec.complianceConfig.upgradesAvailable)"?